web/src/components/tasks/business-formation/review/ReviewNonprofitProvisions.tsx

```diff
@@ -0,0 +1,160 @@
+import { BusinessFormationContext } from "@/contexts/businessFormationContext";
+import { useConfig } from "@/lib/data-hooks/useConfig";
+import { ReactElement, useContext } from "react";
+
+export const ReviewNonprofitProvisions = (): ReactElement => {
+  const { state } = useContext(BusinessFormationContext);
+  const { Config } = useConfig();
+
+  const fillInByLawFormText = (stringToCompare: string | undefined, outputText: string): string => {
+    if (stringToCompare === "IN_BYLAWS") {
+      return `${outputText} ${Config.formation.fields.isVeteranNonprofit.inByLaws}`;
+    }
+
+    if (stringToCompare === "IN_FORM") {
+      return `${outputText} ${Config.formation.fields.isVeteranNonprofit.inForm}`;
+    }
+
+    return `${outputText}:`;
+  };
+
+  const isVisibleInReview = (stringToCompare: string | undefined): boolean => {
+    if (stringToCompare === "IN_FORM") {
+      return true;
+    } else {
+      return false;
+    }
+  };
+
+  const showQuestionAnswer = (questionString: string | undefined, outputText: string): ReactElement => {
+    return (
+      <>
+        <Content style={{ display: "inline-block" }}>
+          {fillInByLawFormText(questionString, outputText)}
+        </Content>
+        {questionString === undefined && (
+          <div style={{ display: "inline-block" }}>
+            &nbsp;
+            <span className={"bg-accent-warm-extra-light text-italic"}>
+              {Config.formation.general.notEntered}
+            </span>
+          </div>
+        )}
+      </>
+    );
+  };
+
+  return (
+    <>
+      <hr className="margin-y-205" />
+      <ReviewSubSection header={"Provisions"}>
+        <div className="margin-top-2">
+          {state.formationFormData.hasNonprofitBoardMembers === undefined &&
+            showQuestionAnswer(
+              state.formationFormData.hasNonprofitBoardMembers,
+              Config.formation.fields.isVeteranNonprofit.undefinedBoardMembersReviewText
+            )}
+          {state.formationFormData.hasNonprofitBoardMembers && (
+            <Content>{Config.formation.fields.isVeteranNonprofit.yesBoardMembersReviewText}</Content>
+          )}
+          {!state.formationFormData.hasNonprofitBoardMembers &&
+            state.formationFormData.hasNonprofitBoardMembers !== undefined && (
+              <Content>{Config.formation.fields.isVeteranNonprofit.noBoardMembersReviewText}</Content>
+            )}
+        </div>
+
+        <div className="margin-top-2 margin-left-3">
+          <ExpandCollapseString
+            text={state.formationFormData.nonprofitBoardMembersTerms}
+            viewMoreText={Config.formation.general.readMoreText}
+            viewLessText={Config.formation.general.readLessText}
+            lines={2}
+          />
+        </div>
+
+        <div className="margin-top-2">
+          {state.formationFormData.hasNonprofitBoardMembers &&
+            state.formationFormData.hasNonprofitBoardMembers !== undefined &&
```

Comment:
[pebble] I think `state.formationFormData.hasNonprofitBoardMembers && state.formationFormData.hasNonprofitBoardMembers !== undefined` is redundant. 

If `state.formationFormData.hasNonprofitBoardMembers` evaluates to truthy then it cannot be undefined, and the second condition is unnecessary.

Reply:
Done